tests/testthat/test-bind.R

```diff
@@ -344,6 +344,20 @@ test_that("can supply `.names_to` to `vec_rbind()` (#229)", {
   expect_identical(vec_rbind(x, b = y, .names_to = "quux")$quux, c("", "", "b"))
 })
 
+test_that("can supply existing `.names_to`", {
+  x <- data.frame(a = 1, id = TRUE)
+  expect_identical(
+    vec_rbind(foo = x, bar = c(a = 2), .names_to = "id"),
```

Comment:
Should mention this in news too. I assume this is consistent with the existing dplyr behaviour?

Reply:
This is not a new behaviour in vctrs, the tests were just missing.

This is not consistent with dplyr behaviour, but it is broken:

```r
x <- data.frame(a = 1, id = TRUE)

dplyr::bind_rows(x, bar = c(a = 2), .id = "id")
#>   id a   id
#> 1  1 1 TRUE
#> 2  2 2   NA
```